### ISSUE TITLE
Clones+ Idea

### ISSUE THREAD

**Issue by UndermanReal** [TurboWarp/extensions#624]
I think there should be a way to detect if a sprite is touching **another** sprite with variables that only belong to that sprite set to certain things. This could be very useful
![image](https://github.com/TurboWarp/extensions/assets/131917462/677ec3bc-a1cf-46e1-a968-9e5835d38e2a)

**Comment by LilyMakesThings** [comment]
Yes I am aware there's not a way to do this in clones+, I've been trying to work on an update for clones+ to amend some of the problems, however it requires a feature that isn't supported fully.

**Comment by LilyMakesThings** [comment]
Planned for over a month now, yes
